crates/bevy_ecs/src/schedule_v3/schedule.rs

```diff
@@ -0,0 +1,1027 @@
+use std::{
+    borrow::Cow,
+    fmt::{Debug, Write},
+    result::Result,
+};
+
+#[cfg(feature = "trace")]
+use bevy_utils::tracing::info_span;
+use bevy_utils::{
+    petgraph::{algo::tarjan_scc, prelude::*},
+    thiserror::Error,
+    tracing::{error, info, warn},
+    HashMap, HashSet,
+};
+
+use fixedbitset::FixedBitSet;
+
+use crate::{
+    self as bevy_ecs,
+    component::ComponentId,
+    schedule_v3::*,
+    system::{BoxedSystem, Resource},
+    world::World,
+};
+
+/// Resource that stores [`Schedule`]s mapped to [`ScheduleLabel`]s.
+#[derive(Default, Resource)]
+pub struct Schedules {
```

Comment:
❓ Why would the system need multiple schedules? Is it for different worlds?

Reply:
In the new design, multiple schedules are used for:

- startup systems
- fixed timestep run criteria
- complex custom logic (like turn based games)
- state-tied systems

I'm not sure this is the right place to document that though.